src/main/java/org/mdbenefits/app/submission/conditions/HasHouseholdMemberDisabledOrOver60.java

```diff
@@ -0,0 +1,44 @@
+package org.mdbenefits.app.submission.conditions;
+
+import static java.util.Collections.emptyList;
+import static org.mdbenefits.app.utils.HouseholdUtilities.isMember60orOlder;
+
+import formflow.library.config.submission.Condition;
+import formflow.library.data.Submission;
+import java.util.List;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+@Component
+public class HasHouseholdMemberDisabledOrOver60 implements Condition {
+
+    @Override
+    public Boolean run(Submission submission) {
+        var inputData = submission.getInputData();
+        List<Map<String, Object>> householdMembers = (List) inputData.getOrDefault("household", emptyList());
```

Comment:
@bseeger - added an empty list as default because this was breaking when household does not exist.